### PR TITLE
Tags/Tag Components

### PR DESCRIPTION
**Pull request by phuson**
- Adds Tags and Tag components as an easy and user-friendly way to quickly generate a list of tags
- Tags component mainly wraps the Box component
- Tag component mainly wraps the Anchor component

### CODE REVIEW

File: src/js/components/Tag.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    if (this.props.className) {
+      classes.push(this.props.className);
+    }
+
+    var other = pick(this.props, keys(Anchor.propTypes));
+
+    return (
+      <div className={classes.join(' ')} onClick={this.props.onClick}>
+        {this.props.children}
+        <Anchor {...other} className="tag--label">
```
Comment (alansouzati) [inline src/js/components/Tag.js]: can we replace `tag--label` by `${CLASS_ROOT}--label`, you can use string template in ES6 which is really cool for that. https://github.com/grommet/grommet/blob/master/src/js/components/Distribution.js#L381

File: src/js/components/Tags.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      classes.push(this.props.className);
+    }
+
+    var other = pick(this.props, keys(Box.propTypes));
+
+    return (
+      <Box {...other}
+        className={classes.join(' ')}
+        direction={this.props.direction ? this.props.direction : 'row'}
+        align={this.props.align ? this.props.align : 'start'}
```
Comment (alansouzati) [inline src/js/components/Tags.js]: can we use default props for both direction and align ?

File: src/js/components/Tag.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import Anchor from './Anchor';
+import pick from 'lodash/object/pick';
+import keys from 'lodash/object/keys';
+
+const CLASS_ROOT = "tag";
+
+export default class Tag extends Component {
+
+  render () {
+    var classes = [CLASS_ROOT];
```
Comment (alansouzati) [inline src/js/components/Tag.js]: as we moved to ES6 would you mind replacing `var` by `let` ?

File: src/js/components/Tag.js (hunk 1 above)
Comment (phuson) [inline src/js/components/Tag.js]: Fixed.

File: src/js/components/Tags.js (hunk 2 above)
Comment (phuson) [inline src/js/components/Tags.js]: Good catch! Fixed.

**Comment by alansouzati** [comment]
Thanks for your contribution. we can review the hover styling in the future.

**Pull request merged**